src/SCRIPTS/TOOLS/bf.lua

```diff
@@ -9,7 +9,7 @@ local scriptsCompiled = assert(loadScript("COMPILE/scripts_compiled.lua"))()
 if scriptsCompiled then
     protocol = assert(loadScript("protocols.lua"))()
     radio = assert(loadScript("radios.lua"))()
-    assert(loadScript(protocol.transport))()
+    assert(loadScript(protocol.mspTransport))()
```

Comment:
Has to be changed here too https://github.com/betaflight/betaflight-tx-lua-scripts/blob/61a860fcdd94ffb92362ae471966f3b17c91046f/src/SCRIPTS/FUNCTIONS/bfbkgd.lua#L4

Reply:
addressed here:
https://github.com/betaflight/betaflight-tx-lua-scripts/blob/7285a6708da4f55759f7e38c0c52cbcd3f349abc/src/SCRIPTS/FUNCTIONS/bfbkgd.lua#L4